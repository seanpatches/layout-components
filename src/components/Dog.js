import React from 'react';

const dog = {
  name: 'spot',
  age: 5,
  weight: '10 lbs'
};

export default function Dog() {
  return (
    <dl>
      <dt>Name</dt>
      <dd>{dog.name}</dd>

      <dt>Age</dt>
      <dd>{dog.age}</dd>

      <dt>Weight</dt>
      <dd>{dog.weight}</dd>
    </dl>
  );
}
