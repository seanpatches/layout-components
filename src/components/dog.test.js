import React from 'react';
import Dog from './Dog';
import { shallow } from 'enzyme';

describe('test dog component', () =>{
  it('produces a dog', () => {
    const wrapper = shallow(<Dog />);
    expect(wrapper).toMatchSnapshot();
  });
});
