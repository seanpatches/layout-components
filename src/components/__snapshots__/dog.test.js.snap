// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test dog component produces a dog 1`] = `
<dl>
  <dt>
    Name
  </dt>
  <dd>
    spot
  </dd>
  <dt>
    Age
  </dt>
  <dd>
    5
  </dd>
  <dt>
    Weight
  </dt>
  <dd>
    10 lbs
  </dd>
</dl>
`;
